### ISSUE TITLE
macOS issue for installing JAABA

### ISSUE THREAD

**Issue by turtlelover426** [kristinbranson/JAABA#340]
I am trying to install JAABA 0.6. on MacOS 11.2.3 with MATLAB 2014a

I am getting an issue that I am not the system administrator and I have taken all steps to resolve this issue on my computer, but it still is not opening the application and displays the same popup. 

I also tried opening JAABA 0.5 with MATLAB 2014a, and it says that the MCR cannot be found for installation.

Please let me know how to resolve this. Thank you.

**Comment by mkabra** [comment]
Hi,

The compiled JAABA application was compiled against versions of MATLAB that
are probably not compatible with current MAC OS versions and probably why
it is not starting up. Instead of the compiled JAABA application, you can
use JAABA from the source in MATLAB.

Mayank

**Comment by turtlelover426** [comment]
Hi Mayank,

Thanks for your reply.

Should I be using a newer version of Matlab? I also downloaded the 0.6
version from the sourceforge.

Would you kindly be able to provide me step by step instructions for this,
as I am very new to both applications.

I am a high schooler using this for a research project. Thanks again.

Sincerely,
Mirika

**Comment by mkabra** [comment]
Yes, use the newer version of Matlab. And use git to clone the JAABA
repository to run using the source code. The instructions are here:
http://jaaba.sourceforge.net/Installation.html#Source

**Comment by turtlelover426** [comment]
Hi Mayank,

I tried cloning the JAABA, and I was able to do so. However, when I go to
run StartJAABA on the terminal command line, it says that "Command not
found"  It is under perframe directory. The path is under
~downloads/JAABA/perframe. The Matlab 2021a is installed at
/applications/MATLAB/Matlab_Runtime/v910 where all the folders are there.
It seems like StartJAABA is not able to find Matlab run time environment.
Happy to get on zoom for 5 minutes if that helps solving this issue.

Thanks.
